docs/_includes/toggle-button.html

```diff
@@ -0,0 +1,492 @@
+<div id="toggle-button">
+    {% include section-header.html name="toggle-button" version=page.versions.toggle-button %}
+
+    <p>The Toggle Button is a button that can be toggled/pressed on or off. It is an atomic piece of the Toggle Button Group.</p>
```

Comment:
We don't have toggle button group yet, so I suggest something like this for now:

```suggestion
    <p>A toggle button is a special type of button that conveys a pressed or non-pressed state; this state is conveyed programmatically via the [aria-pressed](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed) attribute.</p> 
    <p>Toggle buttons can be used in single-select or multi-select groups.</p>
```